### DISCUSSION TITLE
Helm External Database Postgres Version

### DISCUSSION THREAD

**Discussion by aqeelat** [airbytehq/airbyte#40644] [Deployment] [answered]
The docs says "The Airbyte Database only works with Postgres 13."
Why is this a hard dependency? Wouldn't newer Postgres versions be backwards compatible?
Did anyone try using a version newer that 13?

**Comment by marcosmarxm** [comment, marked as answer]
This was updated last week @aqeelat https://github.com/airbytehq/airbyte/pull/39510/files

**Reply by marcosmarxm** [reply]
You can use Postgres 15
